configs/db/postgres/000009_update_operation_retries.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE scheduler_operations
```

Comment:
you can actually do it in step 000008, can't you?

Reply:
Yup. I will do the ddl changes in one step.